src/styles/_cta.scss

```diff
@@ -151,6 +151,26 @@ a.cta {
     opacity: 1;
 }
 
+.transparent-white {
+    background: transparent 0% 0% no-repeat padding-box;
+    box-shadow: 0px 13px 26px #00956341;
+    border: 1px solid #FFFFFF;
+    border-radius: 0%;
+    letter-spacing: 0;
```

Comment:
You can just remove this letter-spacing: 0.

Reply:
removed unnecessary attributes